flask_resty/testing.py

```diff
@@ -27,28 +50,39 @@ def get_meta(response):
     return get_body(response)['meta']
 
 
-def assert_value(actual, expected):
+def assert_similar(actual, expected):
     if isinstance(expected, Mapping):
         assert isinstance(actual, Mapping)
         # Unlike all the others, this checks that the actual items are a
         # superset of the expected items, rather than that they match.
         for key, value in expected.items():
             if value is not UNDEFINED:
                 assert key in actual
-                assert_value(actual[key], value)
+                assert_similar(actual[key], value)
             else:
                 assert key not in actual
     elif isinstance(expected, basestring):
-        assert actual == expected
+        assert expected == actual
```

Comment:
the reason for this is because in python 2, `datetime.now() != JUST_BEFORE`, but `JUST_BEFORE == datetime.now()`, thanks to a crappy python implementation of datetime comparison. fwiw, even `datetime.now() != ANY` which is really bad. python3 works as expected